sdk/src/contexts/solana/context.ts

```diff
@@ -77,6 +78,9 @@ import {
 const SOLANA_SEQ_LOG = 'Program log: Sequence: ';
 const SOLANA_CHAIN_NAME = MAINNET_CONFIG.chains.solana!.key;
 
+// Add priority fee according to 75th percentile of recent fees paid
+const SOLANA_FEE_PERCENTILE = 0.75;
```

Comment:
this smth that should go in an overridable config?

Reply:
definitely could, but I wanna get this change in first. can always do that later